src/pages/find-offers/FindOffers.tsx

```diff
@@ -168,14 +189,27 @@ const FindOffers = () => {
         )}
         {offersLoading ? (
           <Loader pageLoad size={70} />
-        ) : !offers.length && !offersLoading ? (
-          <NotFoundResults description={t('findOffers.notFound.description')} />
         ) : (
-          <OfferContainer
-            isFiltersOpen={isOpen}
-            offerCards={offers}
-            viewMode={cardsView}
-          />
+          <Box
+            onLoad={updateOfferContainerHeight}
+            sx={styles.offerContainerHeight}
+          >
+            {!offers.length && !offersLoading ? (
+              <Box ref={offerContainerRef}>
```

Comment:
You don't need to add two boxes with the same ref.
Do it like this:
<Box
            onLoad={updateOfferContainerHeight}
            sx={styles.offerContainerHeight}
            ref={offerContainerRef}
          >